pkgs/development/tools/misc/micronucleus/default.nix

```
@@ -1,38 +1,35 @@
-{
-  pkgs
-, stdenv
+{ stdenv
 , libusb-compat-0_1
 , fetchFromGitHub
 , lib
 }:
 stdenv.mkDerivation rec {
-   pname = "micronucleus";
-   version = "2.04";
+  pname = "micronucleus";
+  version = "2.04";
 
-   sourceRoot = "source/commandline";
+  sourceRoot = "source/commandline";
 
-   src = fetchFromGitHub {
-     owner = "micronucleus";
-     repo = "micronucleus";
-     rev = version;
-     sha256 = "14msy9amlbflw5mqrbs57b7bby3nsgx43srr7215zyhfdgsla0in";
-   };
+  src = fetchFromGitHub {
+    owner = "micronucleus";
+    repo = "micronucleus";
+    rev = version;
+    sha256 = "14msy9amlbflw5mqrbs57b7bby3nsgx43srr7215zyhfdgsla0in";
+  };
 
-   buildInputs = [ libusb-compat-0_1 ];
-   makeFlags = lib.optionals stdenv.isDarwin [ "CC=cc" ];
+  buildInputs = [ libusb-compat-0_1 ];
+  makeFlags = lib.optionals stdenv.isDarwin [ "CC=cc" ];
 
-   installPhase = ''
-     mkdir -p $out/bin
-     mkdir -p $out/lib/udev
-     cp micronucleus $out/bin
-     cp 49-micronucleus.rules $out/lib/udev
-   '';
-
-   meta = with lib; {
-     description = "Upload tool for micronucleus";
-     homepage = "https://github.com/micronucleus/micronucleus";
-     license = licenses.gpl3;
-     maintainers = [ maintainers.cab404 ];
-   };
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib/udev
+    cp micronucleus $out/bin
+    cp 49-micronucleus.rules $out/lib/udev
+  '';
 
+  meta = with lib; {
+    description = "Upload tool for micronucleus";
+    homepage = "https://github.com/micronucleus/micronucleus";
+    license = licenses.gpl3;
```

The license file in the repo's root [says](https://github.com/micronucleus/micronucleus/blob/master/License.txt) GPLv2 or GPLv3.

The package is only the `commandline` (acc. to line 10). The [header](https://github.com/micronucleus/micronucleus/blob/master/commandline/micronucleus.c) contains the text of the MIT license.

  ```suggestion
    license = licenses.mit;
```

---

Can you open a separate PR with that? I want to get this merged today to save a good amount of time on my darwin builder.